src/components/NewLocationPage/SparkLineBlock/utils.ts

```diff
@@ -101,16 +101,10 @@ export function getSparkLineSeriesFromProjection(
   }));
 }
 
-function getMaxY(series: Column[]) {
-  return d3Max(series, (d: Column) => d.y);
-}
-
-export function getOverallMaxY(seriesA: Column[], seriesB: Column[]) {
-  const maxA = getMaxY(seriesA);
-  const maxB = getMaxY(seriesB);
-  assert(maxA, 'Maximum value unexpectedly not found'); // theres probably a better way
-  assert(maxB, 'Maximum value unexpectedly not found');
-  return Math.max(maxA, maxB);
+export function getMaxY(series: Column[]) {
+  const maxY = d3Max(series, (d: Column) => d.y);
+  assert(maxY, 'Maximum value unexpectedly not found'); // theres probably a better way
```

Comment:
If `maxY` is `0` the assertion will throw, maybe you can use `Number.isFinite` instead?

```ts
assert(Number.isFinite(maxY), '...');
```

Reply:
removed the assertion and added an error boundary